temporal/api/cloud/auditlog/v1/message.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package temporal.api.cloud.auditlog.v1;
+
+option go_package = "go.temporal.io/api/cloud/auditlog/v1;auditlog";
+option java_package = "io.temporal.api.cloud.auditlog.v1";
+option java_multiple_files = true;
+option java_outer_classname = "MessageProto";
+option ruby_package = "Temporalio::Api::Cloud::AuditLog::V1";
+option csharp_namespace = "Temporalio.Api.Cloud.AuditLog.V1";
+
+import "google/protobuf/timestamp.proto";
+
+// LogRecord represents an audit log entry from Temporal, structured for easy parsing and analysis.
+message LogRecord {
```

Comment:
Are all log records a result of an operation? Is there a need to abstract this, e.g. put everything related to an operation in a separate message/field and leave log record generic for other forms of log records later? Having `operation` and `status` here while also having operation details as separate is a bit offputting.

Reply:
Yes all records should be the result of an operation. Status is technically one of `OK` & `ERROR` (could mention this but it's in docs already fwiw)

Reply:
Hrmm, I fear that baking in that forced assumption may harm reusability of this log record for other auditable things. I would encourage anything operation specific to be in its own field. It is confusing that half operation-specific things are top-level and half in details.

Reply:
Based on internal discussion, we are not allowed to change structure in this review.